tests/testpkgs/default.nix

```diff
@@ -4,6 +4,7 @@
   bitbucket-snapshot = pkgs.callPackage ./bitbucket.nix { isSnapshot = true; };
   cargoLock.expand = pkgs.callPackage ./cargo-lock-expand { };
   cargoLock.update = pkgs.callPackage ./cargo-lock-update { };
+  composer.update = pkgs.callPackage ./composer.nix { };
```

Comment:
```suggestion
  composer = pkgs.callPackage ./composer.nix { };
```

Reply:
The test is updating "composer" and not "composer.update"

Reply:
Oh thanks!